.github/workflows/acctest.yml

```diff
@@ -69,3 +69,31 @@ jobs:
         env:
           TF_ACC_TERRAFORM_VERSION: ${{ matrix.terraform }}
         run: make testacc
+
+  lint:
+    name: Lint
+    needs: build
+    runs-on: ubuntu-latest
+    container: golangci/golangci-lint:v1.33.0
```

Comment:
golangci-lint recommends a separate workflow. Any reason to combine them?

Reply:
I see - we are using `make lint`, not their GH Action.

Reply:
Actually, this PR adds a config file, so we could as well use the GH Action.

Reply:
https://github.com/golangci/golangci-lint-action

> We recommend running this action in a job separate from other jobs (go test, etc) because different jobs run in parallel.

Might as well use their GH action, even if the limitations don't apply.

Reply:
Yeah, that's what I meant. Done.